paprika/build.py

```diff
@@ -149,19 +209,7 @@ def solvate(tleapfile, pdbfile=None, pbctype=1, bufferwater='12.0A',
         dir = '.'
 
     # Read tleapfile
-    tleaplines = []
-    with open(tleapfile, 'r') as f:
-        for line in f.readlines():
-            if re.search('loadpdb', line):
-                words = line.rstrip().replace('=', ' ').split()
-                if pdbfile is None:
-                    pdbfile = words[2]
-                unit = words[0]
-                tleaplines.append(
-                    "{} = loadpdb {}\n".format(unit, pdbfile))
-            if not re.search(r"^\s*addions|^\s*addions2|^\s*addionsrand|^\s*desc|"
-                             r"^\s*quit|^\s*solvate|loadpdb|^\s*save", line, re.IGNORECASE):
-                tleaplines.append(line)
+    tleaplines = read_tleaplines(tleapfile, pdbfile=pdbfile, skip_solvate=True)
```

Comment:
To make things simpler, I imagine `read_tleaplines` should just take a `tleap` output file as argument -- unless this is also running `tleap` here? If so, probably should clarify function names a bit.

Reply:
happy to take suggests on naming.

`read_tleaplines` reads the text of a user's tleap file, and modifies the loadpdb statement if requested, then returns the text as a list.